### ISSUE TITLE
Lens throws error on JATS document with no <back>

### ISSUE THREAD

**Issue by pkra** [elifesciences/lens#145]
Seems like https://github.com/elifesciences/lens/blob/ams-fixes/converter/lens_converter.js#L959 should check if `back` exists before calling `Article.Back`.

**Comment by pkra** [comment]
Note, this is in the ams-fixes branch.

**Comment by michael** [comment]
:+1:

**Comment by CAYdenberg** [comment]
Can you close this @pkra ?

**Comment by pkra** [comment]
> Can you close this @pkra ?

This should be closed automatically once the code is merged into `master`. Not sure what the policy for closing issues is though, so if @michael and @oliver---- want to close it, that's fine by me.
